python/dgl/distributed/partition.py

```diff
@@ -24,6 +34,43 @@
     ETYPE: F.int32
     }
 
+def _format_part_metadata(part_metadata, formator):
```

Comment:
formator -> formatter?

Not sure whether formator is a special word in dgl, if you are talking about data-formatting, it should be formatter.